roles/cloud_setup/tasks/cleanup.yml

```diff
@@ -0,0 +1,62 @@
+---
+# This task cleans up the unwanted files to free up disk space after the
+# installation process is complete, it also creates the required symlinks
+# from the reference data hosted by Galaxy project via CVMFS to the folder
+# `/galaxy/data` on the remote target.
+- name: Clean apt cache to recover some disk space
```

Comment:
I think this is universally useful, we can put this and the autoremove as a post-task in the main playbook.